Pocket/pocket-ios
# Home load performance

## Summary
This PR reduces the time it takes to load content for Home (i.e. the slate lineup). To achieve this improvement, we exclude the reader/offline content from the query. We fetch the details for a recommendation when a user selects the item and the reader is presented.

## References 
https://getpocket.atlassian.net/browse/IN-584

## Test Steps
* Launch app on home tab
* Notice the load time has improved

* Navigate to home tab
* Select a recommendation that is presented in the reader view (not web view)
* Notice the skeleton/loading screen is visible shortly while content is fetched from the server.

## PR Checklist:

## Screenshots
![loading-screen-iphone](https://user-images.githubusercontent.com/323541/182471746-316db7a2-5f50-46d2-9791-95be2a8261b0.png)
![loading-screen-wide](https://user-images.githubusercontent.com/323541/182471657-8daf7522-7c0a-4024-8ae2-610ef9a0b1e8.png)
